frontend/public/components/secret.jsx

```diff
@@ -135,7 +135,7 @@ const SecretsPage = props => {
 
   const createProps = {
     items: createItems,
-    createLink: (type) => `/k8s/ns/${props.namespace}/secrets/new/${type !== 'yaml' ? type : ''}`
+    createLink: (type) => `/k8s/ns/${_.get(props, 'namespace', 'default')}/secrets/new/${type !== 'yaml' ? type : ''}`
```

Comment:
nit: `props.namespace || 'default'` is slightly shorter and handles more values like null and empty string